presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/APIFileHandler.java

```diff
@@ -81,10 +84,16 @@ public Response getFile(String path)
     public Response replaceFileFromURL(String path, String url)
     {
         try {
-            AgentFileUtils.replaceFile(Paths.get(baseDir.toString(), path), url);
-            LOGGER.debug("Replacing file '%s' with url '%s' in progress", path, url);
+            AgentFileUtils.replaceFile(Paths.get(baseDir.toString(), path),
+                    new URI(url).toURL());
+            LOGGER.debug("Replaced file '%s' with file at url '%s'", path, url);
             return Response.status(Status.ACCEPTED)
```

Comment:
Ditto for below. I have done refactoring once for all before to make them either return 200/202 then have the log message to comply with them.